docs/vercel/src/lib/AsciiPreview.svelte

```diff
@@ -35,7 +35,7 @@
 <div class="title-row">
   <div class="language-name">
     <Chip id={name} color={chip} width={24} height={24} />
-    <h3>{name}</h3>
+    <h3 id={name}><a href="#{name}" class="title-link">{name}</a></h3>
```

Comment:
Now that I think about it, a title link could probably be a component.
```svelte
<!-- TitleLink.svelte -->
<script lang="ts">
export let name: string;
</script>

<h3 id={name}<a href="#{name}" class="title-link">{name}</a></h3>

<style>
.title-link,
.title-link:hover,
.title-link:visited {
  color: #222;
}
</script>
```
```svelte
<!-- AsciiPreview.svelte -->
<!-- ... -->
<TitleLink {name} />
```

Reply:
Tbh, it's probably overthinking to separate it :pray:

Reply:
You're probably right. I'm a bit over-obsessed with how git diffs potentially look :laughing: I think the only gain I was thinking of is that, if updating this title (e.g. slugifying the `id`) the diff would be prettier if this was already a separate component.

So, yeah, as I said these changes look good to me. That suggestion is just a matter of opinion, and I'll leave it up to you if you want to implement it or not :slightly_smiling_face:

Reply:
https://github.com/o2sh/onefetch/pull/922/commits/ca5b1ba94af8c034ba5d018bd412070a59e51e76 :smile: